doc/source/user_guide/getting_help.rst

```diff
@@ -26,6 +26,16 @@ Use the ``quick search`` field in the navigation bar of the online
 documentation to find mentions of keywords (segmentation,
 rescaling, denoising, etc.) in the documentation.
 
+
+API Discovery
+-------------
+
+We provide a ``lookfor`` function to search API functions::
+
+   import skimage as ski
+   ski.util.lookfor('eigenvalue')
```

Comment:
Note the change from `ski.lookfor` to `ski.util.lookfor`. It's still available as the former, but not advertised. Now, `ski.util.lookfor` is also included in the HTML API doc.